frontend/ui/bidi.lua

```diff
@@ -0,0 +1,157 @@
+--[[--
+Bidirectional text and UI mirroring helpers.
+Some overview at:
+https://material.io/design/usability/bidirectionality.html
+]]
+
+local Language = require("ui/language")
+local _ = require("gettext")
+
+local Bidi = {
+    _mirrored_ui_layout = false,
+    _rtl_ui_text = false,
+}
+
+-- Setup UI mirroring and RTL text for UI language
+function Bidi.setup(lang)
+    local is_rtl = Language:isLanguageRTL(lang)
+    -- Mirror UI if language is RTL
+    Bidi._mirrored_ui_layout = is_rtl
+    -- Unless requested not to (or requested mirroring with LTR language)
+    if G_reader_settings:isTrue("reverse_lang_ui_layout_mirroring") then
```

Comment:
What about something like `ui_direction_rtl`?

Reply:
To make things less ambiguous, I tried to always use "rtl" for text, and "mirrored" for UI layout.
These 2 things are quite orthogonal to each other (in the implementation, even if in the real world, they should be either both enabled or both disabled).
I'd like to keep that as separate as possible, so we can all distinquish RTL issues (text, text aligmnent) and Mirroring issues (widgets, gestures...), so also for the settings names.